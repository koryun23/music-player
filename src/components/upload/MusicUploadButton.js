import React from "react";
import "../../css/upload/MusicUploadForm.css";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faAdd, faPlus } from "@fortawesome/free-solid-svg-icons";

export default function MusicUploadButton(props) {
    return (
        <button className="music-upload-button">
            <FontAwesomeIcon icon={faPlus} size="lg"/>
        </button>   
    );
}