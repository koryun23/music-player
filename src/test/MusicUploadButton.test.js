import renderer from "react-test-renderer";
import MusicUploadButton from "../components/upload/MusicUploadButton";

it('music upload button renders correctly', () => {
    const tree = renderer.create(<MusicUploadButton />).toJSON();
    expect(tree).toMatchSnapshot();
})
