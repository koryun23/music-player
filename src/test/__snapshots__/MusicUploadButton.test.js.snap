// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`music upload button renders correctly 1`] = `
<button
  className="music-upload-button"
>
  <svg
    aria-hidden="true"
    className="svg-inline--fa fa-plus fa-lg "
    data-icon="plus"
    data-prefix="fas"
    focusable="false"
    role="img"
    style={Object {}}
    viewBox="0 0 448 512"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M256 80c0-17.7-14.3-32-32-32s-32 14.3-32 32V224H48c-17.7 0-32 14.3-32 32s14.3 32 32 32H192V432c0 17.7 14.3 32 32 32s32-14.3 32-32V288H400c17.7 0 32-14.3 32-32s-14.3-32-32-32H256V80z"
      fill="currentColor"
      style={Object {}}
    />
  </svg>
</button>
`;
